pkgs/development/tools/glpaper/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchhg
+, meson, ninja
+, pkg-config
+, wlroots, wayland, wayland-protocols
+, pixman, libxkbcommon
+,libX11
+, libGL, libglvnd
+}:
+
+stdenv.mkDerivation {
+  name = "glpaper";
+  version = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/glpaper";
+    rev = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+    sha256 = "04y12910wvhy4aqx2sa63dy9l6nbs7b77yqpdhc96x2b3mgzgjfs";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config ];
+  buildInputs = [
+    wlroots wayland wayland-protocols wlroots
+    pixman libxkbcommon
```

Comment:
Why does this need all these dependencies? I only see wayland-client, wayland-egl and egl in https://hg.sr.ht/~scoopta/glpaper/browse/meson.build?rev=default#L5

Reply:
Yeah, I took that from overlay, did not test.

After testing removed :  pixman libxkbcommon libglvd.